Exercise/compute_lowest_terms.py

```diff
@@ -1,4 +1,19 @@
 ## TODO: complete the function "lowest_terms" below
+from math import gcd 
+
 
 def lowest_terms(x):
-	return ""
```

Comment:
write your function without using gcd or any other imports

use only the python built-in modules (no imports allowed)